cypress/integration/models/stakeholdergroups.ts

```diff
@@ -15,34 +26,47 @@ export class Stakeholdergroups {
     }
 
     protected fillName(name: string): void {
-        inputText(stakeholdergroupName, name);
+        inputText(stakeholdergroupNameInput, name);
     }
 
     protected fillDescription(description: string): void {
-        inputText(stakeholdergroupDescription, description);
+        inputText(stakeholdergroupDescriptionInput, description);
+    }
+
+    protected selectMember(member: string): void {
+        selectFormItems(stakeholdergroupMemberSelect, member);
     }
 
     getStakeholdergroupName(): string {
         this.stakeholdergroupName = faker.company.companyName();
         return this.stakeholdergroupName;
     }
 
+    getNewStakeholdergroupName(): string {
+        return faker.company.companyName();
+    }
+
     getStakeholdergroupDescription(): string {
         this.stakeholdergroupDescription = faker.lorem.sentence();
         return this.stakeholdergroupDescription;
     }
 
-    create(): void {
+    create(name?: string, description?: string, member?: string): void {
         Stakeholdergroups.clickStakeholdergroups();
         clickByText("button", "Create new");
         this.getStakeholdergroupName();
         this.getStakeholdergroupDescription();
+        this.stakeholdergroupName = name || this.stakeholdergroupName;
```

Comment:
Not sure about concept here, but you can accomplish an external provider by using default params in the function.

```create(name: string = this.stakeholdergroupName, description:string = this.stakeholdergroupDescription, member?:string): void {
    
}```

Reply:
It is not useful in my case. I want to edit stakeholder group name (with updated name). Hence, I am asking test case to send name value and initializing it. So that, stakeholdergroup.delete() will not need new name to send for deletion.

Reply:
It is the same thing, what you are doing. A default parameter is one, which is used only in case your function call does not pass a param explicitly.
So, that is what we want, that our edit function should be able to get updated name from test.
Therefore, if you use a default param, the edit function will accept the updated name passed to function from test file and at same time, if you don't want to pass value explicitly, initialize it and function will pick up that class variable value automatically. 
And you can get rid of this || thing